lib/connectionConfig.js

```diff
@@ -503,6 +503,15 @@ ConnectionConfig.prototype.pullURL = function () {
   return purl;
 };
 
+/**
+ * Generate URL for diff endpoint
+ * @returns {string}
+ */
+ConnectionConfig.prototype.diffURL = function () {
+  const purl = this.branchBase('diff');
+  return purl;
```

Comment:
For next time, you can save one line of code by doing: `return this.branchBase('diff');`